app/src/main/java/net/gsantner/opoc/util/ShareUtil.java

```diff
@@ -1136,7 +1136,7 @@ public void writeFile(final File file, final boolean isDirectory, final Callback
                 if (isDirectory) {
                     file.mkdirs();
                 } else {
-                    fileOutputStream = new FileOutputStream(file);
+                    fileOutputStream = new FileOutputStream(file, true);
```

Comment:
Adding append here seems to solve the overwrite issue

Reply:
Needs more testing

Reply:
Description of java append is: `    append - if true, then bytes will be written to the end of the file rather than the beginning`. I would say this is not the right way.